packages/api-specification/interface.ts

```diff
@@ -117,6 +117,59 @@ declare namespace ssf {
     transparent?: boolean;
   }
 
+  class EventEmitter {
+    /**
+     * Adds a lister that runs when the specified event occurs. Alias for <span class="code">on()</span>.
```

Comment:
Should this be "listener" rather than "lister"? (same typo in the other function docs too)